### PR TITLE
Release 2.18 screenshots

### PR DESCRIPTION
**Pull request by dsarkar**
Internal Tracking ID: [EXPOSUREAPP-11663](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11663)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@dsarkar 
As mentioned by @Marcono1234 relating to older screenshot versions, there are mistakes in the screenshot numbering in the `alt` texts. For example:

https://github.com/corona-warn-app/cwa-website/blob/4f5252fe9ebb8f50c2e73b787b841847ba76b70e/src/data/screenshots.json#L121-L122

https://github.com/corona-warn-app/cwa-website/blob/4f5252fe9ebb8f50c2e73b787b841847ba76b70e/src/data/screenshots.json#L163-L165

and so on through the whole file.

**Comment by Marcono1234** [comment]
I have mentioned this in #2453, but my point was to not fix them manually (see description of that issue).

**Comment by MikeMcC399** [comment]
@Marcono1234 
> I have mentioned this in https://github.com/corona-warn-app/cwa-website/issues/2453, but my point was to not fix them manually (see description of that issue).

- The most reliable way to fix the issue is to follow your good suggestion in #2453. The short-term way to fix the 2.18 files which are due to be released in 2 days is to correct them manually.

**Comment by MikeMcC399** [comment]
@larswmh 
Thank you for correcting the alt text numbering!

File: src/data/screenshots_de.json [hunk 1]
```diff
@@ -5,17 +5,14 @@
...
+                "<b>Zentrale Verwaltung von Regeln:</b> Die Regeln für Zertifikate (z.&nbsp;B. hinsichtlich ihrer Gültigkeit, oder Priorisierung, wenn mehrere Zertifikate vorhanden sind) und für die Ermittlung des G-Status wurden aus der Corona-Warn-App herausgelöst. Diese Regeln sind jetzt zentral abgelegt und können daher unabhängig von den App-Releases angepasst werden. Damit kann schneller auf Regeländerungen reagiert werden. Nach einer Änderung werden die aktualisierten Regeln an die App übertragen. Für Sie ändert sich an der Benutzung der Corona-Warn-App nichts durch diese zentrale Verwaltung der Regeln.",
+                "<b>Anpassung der Darstellung für Auffrischungsimpfung:</b> Im Rahmen der Erinnerung für Auffrischungsimpfungen gibt es nun eine übersichtliche Darstellung. Weitere Informationen zu einer empfohlenen Auffrischungsimpfung sind nun durch das Tippen auf das Feld “Hinweis zur Auffrischimpfung” einsehbar. Siehe Screenshots 'Hinweis zur Auffrischimpfung' für sowohl <a href='#android_booster_note' target='_blank'>Android</a> als auch <a href='#ios_booster_note' target='_blank'>iOS</a> ",
```
Comment (MikeMcC399) [inline src/data/screenshots_de.json]: Auffrischungsimpfung => Auffrischimpfung

See issue https://github.com/corona-warn-app/cwa-website/issues/2162 from @svengabr

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
- RKI requested "Auffrischungsimpfung"  to be written as "Auffrischimpfung" according to issue #2162.

I can't work out why my in-line comment is shown as "Outdated". 

https://github.com/corona-warn-app/cwa-website/blob/c821c035b0cf45bc05a674c0fe254f82a9e813c6/src/data/screenshots_de.json#L15

shows the text I'm referring to.

File: src/data/screenshots_de.json (hunk 1 above)
Comment (MikeMcC399) [inline src/data/screenshots_de.json]: Auffrischungsimpfung => Auffrischimpfung

Comment (larswmh) [reply src/data/screenshots_de.json]: Fixed it in commit [8eb7705](https://github.com/corona-warn-app/cwa-website/pull/2452/commits/8eb7705727bcb144955f83c075361327834dd115), thanks!

Comment (larswmh) [reply src/data/screenshots_de.json]: Fixed it in commit [8eb7705](https://github.com/corona-warn-app/cwa-website/pull/2452/commits/8eb7705727bcb144955f83c075361327834dd115), thanks!

**Review by MikeMcC399 — APPROVED** [review]
@larswmh 
Thanks for taking care of my comments!

**Comment by dsarkar** [comment]
@MikeMcC399 Many thanks for reviewing!

**Pull request merged**